integration/all/integration_test.go

```diff
@@ -0,0 +1,314 @@
+// Copyright 2019-2022 The Inspektor Gadget authors
```

Comment:
Right, I did ask @mauriciovasquezbernal about this, and he suggested that we should have those tests in `integration/inspektor-gadget`.
But you are right, it would remove a lot of duplication.

Reply:
I think we can put all integration tests in the same folder (perhaps in `/integration/`) and skip kubectl-gadget or ig specific as suggested by @blanquicet above.